README.md

```diff
@@ -547,6 +547,23 @@ sent via this stream.
   client.stop_stream(cancel_requests=True)
 ```
 
+For AsyncIO requests, an AsyncIO task wrapping an `infer()` coroutine can be
+safely cancelled.
+
+```python
+  infer_task = asyncio.create_task(aio_client.infer())
+  await something_else
+  infer_task.cancel()
+```
+
+For AsyncIO streaming requests, `cancel()` can be called on the asynchronous
+iterator returned by `stream_infer()` API.
+
+```python
+  responses_iterator = aio_client.stream_infer()
+  responses_iterator.cancel()
+```
+
 See more details about these APIs in
```

Comment:
L567-L568 discuss about non-aio client library. Can you move these lines over L550?

And add few lines here pointing to the grpc asyncio version?

Reply:
Sure, updated.